Alerting: PagerDuty: adding current state to the payload (#29270)
grafana/grafana: docs/sources/alerting/notifications.md

This behavior will become the default in a future version of Grafana.

> > Using `dedup_key` tag will override Grafana generated `dedup_key` with a custom key.

Reviewer: ```suggestion
> **Note:** The `dedup_key` tag overrides the Grafana-generated `dedup_key` with a custom key.
```

> > Using `state` tag will override current alert state inside `custom_details` payload.

Reviewer: ```suggestion
> **Note:** The `state` tag overrides the current alert state inside the `custom_details` payload.
```
